project/Dependencies.scala

```diff
@@ -119,7 +119,7 @@ object Dependencies {
       // If changed, update akka-docs/build.sbt as well
       val sigarLoader  = "io.kamon"         % "sigar-loader"        % "1.6.6-rev002"     %     "optional;provided;test" // ApacheV2
 
-      val levelDB       = "org.iq80.leveldb"            % "leveldb"          % "0.7"    %  "optional;provided"     // ApacheV2
+      val levelDB       = "org.iq80.leveldb"            % "leveldb"          % "0.9"    %  "optional;provided"     // ApacheV2
```

Comment:
please update in this documentation file also: https://raw.githubusercontent.com/akka/akka/master/akka-docs/src/main/paradox/scala/persistence.md

Reply:
Done.